tests/unittests/api/helpers/test_storage.py

```diff
@@ -0,0 +1,34 @@
+"""Test file for storage functions."""
+
+from app.api.helpers.storage import create_url
+from tests.unittests.setup_database import Setup
+from tests.unittests.utils import OpenEventTestCase
+
+
+class TestStorageHelperValidation(OpenEventTestCase):
+    """Test class for testing storage helper functions."""
+
+    def setUp(self):
+        """Method to set up app for testing."""
+        self.app = Setup.create_app()
+
+    def test_create_url(self):
```

Comment:
One test should handle one URL. Create 3 different tests to check

- http 80
- https 443
- arbitrary port